terraform/storage/main.tf

```diff
@@ -0,0 +1,12 @@
+resource "google_storage_bucket" "default" {
```

Comment:
perhaps we can call it something more descriptive like 'bucket' ? Makes it more readable when we need to reference it in other bits of code

Reply:
hm I did this intentionally thinking a module's resources should never be referenced outside of the module itself? only in the module's output (like I did in `output.tf`) or is that incorrect?

Reply:
not outside the module no but the modules might get more complex over time. Anyway, not important for now.